gdi/get-data-in/application/go/splunk-go-otel-distribution.rst

```diff
@@ -9,7 +9,7 @@ About the Splunk Distribution of OpenTelemetry Go
 
 The Splunk Distribution of OpenTelemetry Go extends the OpenTelemetry Instrumentation for Go, a set of open source Go packages that provides APIs to directly measure the performance and behavior of your software.
 
-With the distribution you can instrument any application written in Go 1.19 and higher. See :ref:`go-otel-requirements`.
+With the distribution you can instrument any application written in Go 1.22 and higher. See :ref:`go-otel-requirements`.
```

Comment:
@pellared Do we also need to update:

https://github.com/splunk/public-o11y-docs/blob/d9cba9f515ca2d863a6def9d25ab9355da911c02/_includes/requirements/go.rst?plain=1#L1

Reply:
https://github.com/splunk/public-o11y-docs/pull/1521/commits/26040b329ed0ab36d4197cb941e78ac1cb00cf99